pkg/apis/core/v1beta1/defaults.go

```diff
@@ -321,6 +321,34 @@ func SetDefaults_Worker(obj *Worker) {
 	}
 }
 
+// SetDefaults_ClusterAutoscaler sets default values for ClusterAutoscaler object.
+func SetDefaults_ClusterAutoscaler(obj *ClusterAutoscaler) {
```

Comment:
I have no hard opinion, but defaulting already existing fields should be done in separate PR.

Reply:
It's a good point @vpnachev. I will keep it in mind for future reference. However, for this change, I would like to go with this single PR/commit. I hope it's alright.